manifests/config/global/jenkins_config.pp

```diff
@@ -0,0 +1,24 @@
+# This private class is called from jenkins::config to manage Jenkins' config.xml
+#
+class jenkins::config::global::jenkins_config(
```

Comment:
This class name seems unnecessarily long.  Perhaps `Jenkins::config::global` or `...::core` or `...::main`?